april_vision/frame_sources.py

```diff
@@ -5,19 +5,23 @@
 from typing import List, Optional, Tuple, Union
 
 import cv2
+import numpy as np
 from numpy.typing import NDArray
 
 LOGGER = logging.getLogger(__name__)
 
 
+FrameArray = NDArray[np.generic]
```

Comment:
Facading the numpy array nature of the frames obscures the fact that they are in fact arrays and standard array and opencv methods can be used on them.